content/livesync/connector/monitor.textile

```diff
@@ -1,26 +1,27 @@
 ---
-title: Monitoring
+title: Monitor
+meta_description: "Monitor the health and metrics of the Ably Database Connector."
 product: livesync
 ---
 
 The Database Connector exposes an HTTP server that can be used to monitor instance health and metrics.
 
-The server listens on the port defined by the @ADBC_HEALTH_ADDRESS@ config and defaults to port 2259.
+The server listens on the port defined by the @ADBC_HEALTH_ADDRESS@ "configuration option":/LINK_CONFIGURE and defaults to port @2259@.
```

Comment:
I know nothing about textile, but is `LINK_CONFIGURE` defined?

Reply:
Fixed, thanks.